internal/driver/gomobile/canvas.go

```diff
@@ -382,6 +406,7 @@ func (c *mobileCanvas) tapUp(pos fyne.Position, tapID int,
 
 	if c.menu != nil && c.overlays.Top() == nil && pos.X > c.menu.Size().Width {
 		c.menu.Hide()
+		c.menu.Refresh()
```

Comment:
Hide() doesn't refresh already?

Reply:
It normally would, yes.
However this is slightly complicated by the c.menu = nil below...
Because the menu is an object root it being removed means that it is not traversed and the object is not found in the normal refresh cycle - so I had to call the Refresh().

Alternative is to remove the nil and set it after the refresh completes, but that was more complex.

Reply:
Okay - Good enough reasoning for me.